packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VideoController/Slider.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Grommet, Box, RangeInput } from 'grommet'
+
+const customThemeRangeInput = {
+  global: {
+    spacing: '16px'
+  },
+  rangeInput: {
+    track: {
+      color: 'accent-2',
+      height: '6px',
+      extend: () => `border-radius: 10px`,
+      lower: {
+        color: '#EFF2F5'
+      },
+      upper: {
+        color: 'dark-4'
+      }
+    },
+    thumb: {
+      color: '#EFF2F5'
+    }
+  }
+}
+
+const Slider = ({ played, onMouseUp, onMouseDown, onChange }) => (
+  <Grommet theme={customThemeRangeInput}>
+    <Box a11yTitle='Video player slider'>
```

Comment:
This applies the accessible label to a `div`. Sectioning elements in the DOM won't generally be announced by a screenreader, except for named landmarks like `<header>`, `<nav>`, `<main>` and `<footer>`. I think you want to apply the accessible name to the range input itself, in which case it will be used as the label for that input.

Small wording change: maybe call it 'Video player scrubber' or similar, since the slider is being used as a video scrubber here.

Reply:
Sorry, one other thought. You could move these accessible labels into locale files, in which case they could be made available in other languages at some future point.